crates/wash-cli/src/up/config.rs

```diff
@@ -31,13 +31,15 @@ pub const WASMCLOUD_RPC_JWT: &str = "WASMCLOUD_RPC_JWT";
 pub const WASMCLOUD_RPC_SEED: &str = "WASMCLOUD_RPC_SEED";
 pub const WASMCLOUD_RPC_CREDSFILE: &str = "WASMCLOUD_RPC_CREDSFILE";
 pub const WASMCLOUD_RPC_TLS: &str = "WASMCLOUD_RPC_TLS";
+pub const WASMCLOUD_RPC_TLS_CA_FILE: &str = "WASMCLOUD_RPC_TLS_CA_FILE";
 // NATS CTL connection configuration
 pub const WASMCLOUD_CTL_HOST: &str = "WASMCLOUD_CTL_HOST";
 pub const WASMCLOUD_CTL_PORT: &str = "WASMCLOUD_CTL_PORT";
 pub const WASMCLOUD_CTL_SEED: &str = "WASMCLOUD_CTL_SEED";
 pub const WASMCLOUD_CTL_JWT: &str = "WASMCLOUD_CTL_JWT";
 pub const WASMCLOUD_CTL_CREDSFILE: &str = "WASMCLOUD_CTL_CREDSFILE";
 pub const WASMCLOUD_CTL_TLS: &str = "WASMCLOUD_CTL_TLS";
+pub const WASMCLOUD_CTL_TLS_CA_FILE: &str = "WASMCLOUD_CTL_TLS_CA_FILE";
```

Comment:
Nit: these env vars use "TLS_CA_FILE" but the args just use `-ca-file`. Can we have "tls" in both or neither?